aiohttp/helpers.py

```diff
@@ -356,14 +357,40 @@ def parse_mimetype(mimetype: str) -> MimeType:
     )
 
 
+class EnsureOctetStream(Message):
+    def __init__(self) -> None:
+        super().__init__()
+        self.set_default_type("application/octet-stream")
+
+    def get_content_type(self) -> Any:
+        """Re-implementation from Message
+
+        Returns application/octet-stream in place of plain/text when
+        value is wrong.
+
+        The way this class is used guarantees that content-type will
+        be present so simplify the checks wrt to the base implementation.
+        """
+        value = str(self.get("content-type")).lower()
+
+        # Based on the implementation of _splitparam in the standard library
+        ctype, sep, _ = value.partition(";")
+        if not sep:
+            ctype = ctype.strip()
```

Comment:
This should be done unconditionally (I think you maybe misread the original code):
```suggestion
        ctype = ctype.strip()
```

Reply:
You're correct, I did misread that part.